libnavigation-base/src/test/java/com/mapbox/navigation/base/internal/route/NavigationRouteExTest.kt

```diff
@@ -318,6 +354,17 @@ class NavigationRouteExTest {
             .build(),
     )
 
+    private fun provideDefaultClosures(): List<Closure> = listOf(
+        Closure.builder()
+            .geometryIndexStart(0)
+            .geometryIndexEnd(5)
+            .build(),
+        Closure.builder()
+            .geometryIndexStart(10)
+            .geometryIndexEnd(12)
+            .build(),
```

Comment:
Maybe?
```suggestion
        createClosure(),
        createClosure()
```

Reply:
I need to add the testing base module for it. Let's keep this, as it's the same